src/bookmarks/service/hooks/useBookmarkAddHandler.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  ClientBookmarkCategoryItem,
+  useGETCategoryListQuery,
+} from '@/bookmarks/api/bookmark';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+export type DisClosureType = 'PUBLIC' | 'PRIVATE' | 'FRIENDS';
+
+const useBookmarkAddHandler = () => {
+  // SERVER
+  const { data, isLoading } = useGETCategoryListQuery({
+    userId: '1',
+  });
+  const [categoryList, setCategoryList] = useState<
+    ClientBookmarkCategoryItem[] | undefined
+  >(data);
+  const [selectedCategory, setCategory] = useState('');
```

Comment:
```suggestion
  const [selectedCategoryId, setSelectedCategoryId] = useState('');
```
p2. 이렇게 선택한 카테고리 id라는걸 더 명시해주면 어떨까용??